speedscale/README.md

```diff
@@ -0,0 +1,58 @@
+# Agent Check: Speedscale
+
+## Overview
+
+This integration publishes traffic replay results from [Speedscale][1] into Datadog. This lets you combine your observability data from Datadog with the results of a particular replay to drill-down into the root cause of poor performance. Find and troubleshoot potential performance issues before they show up in production with a combination of Speedscale and Datadog.
+
+## Setup
+
+### Configuration
+
+In order to utilize the integration you need to capture the following from Datadog:
+
+* [API Key][2] - An API key is required by the Datadog Agent to submit metrics and events to Datadog
+
+A best practice is to save this into an environment variable. Most likely you will store this environment variable in your continuous integration system, but when doing a one-off test you can access in your terminal like so:
+
+```
+export DDOG_API_KEY=0
+```
+
+Gather the report id of a specific report that you want to upload into Datadog. When working with continuous integration you should be able to get the report id associated with your commit hash. Store this report id in an environment variable:
+
+```
+export SPD_REPORT_ID=0
+```
+
+Now with the specific report id and the Datadog API key, you run the `speedctl` CLI to export that traffic replay report as a Datadog event.
+
+```
+speedctl export datadog report ${SPD_REPORT_ID} --apiKey ${DDOG_API_KEY}
+✔ {"status":"ok",...}
+```
```

Comment:
indent three spaces. Also, the check mark is causing an error in the file because it is non-ASCII. Is that line part of the command? or what the command returns (if this, then just delete the line).